src/components/MissionAPP/MissionShelf.jsx

```diff
@@ -0,0 +1,49 @@
+import { useDispatch, useSelector } from 'react-redux';
+import { useEffect } from 'react';
+import { getMissions, selectMission } from 'features/missions/missionsSlice';
+import { nanoid } from '@reduxjs/toolkit';
+
+const MissionShelf = () => {
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(getMissions());
+  }, []);
```

Comment:
- [ ] Actually, the `getMissions()` action is dispatched twice, and every time that we change the view using navigation. I guess it is required to only dispatch once (i.e. when changing views / using navigation).
![image](https://github.com/lordksix/space-travelers-hub/assets/122116375/8e559ec0-9d1b-41b0-859a-a2dcef73d849)

Reply:
The app rendered twice because it is on restrict mode.